learing_journal/templates/new-entry.jinja2

```diff
@@ -15,6 +15,7 @@
       <br/>
       <label for="title">Journal Text</label><br/>
       <textarea class="form-control" rows="15" id="body" name="body" placeholder="Enter your new journal text."></textarea>
+      <br/>
```

Comment:
almost every time I see a `<br />` tag in HTML it's because someone needs to fix some CSS.  If the only purpose of this tag is to gain a bit of clearance above the submit button, think again about how you might do that using style rules.